packages/tokens/vite.config.ts

```diff
@@ -0,0 +1,37 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { resolve } from 'node:path';
+
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import dts from 'vite-plugin-dts';
+
+export default defineConfig({
+    plugins: [
+        react(),
+        dts({
+            rollupTypes: true,
+            include: ['src/components/ThemeProvider'],
```

Comment:
Why do you need an array with just that file? The TSConfig should be already taking care of it no?
```suggestion
```